README.md

```diff
@@ -4,26 +4,70 @@
-on that, check [this file](./docs/concepts.rst).
+on that, check [Device Manager Concepts page](./docs/concepts.rst).
+
+**Before running all those steps a Kafka and a Postgres database must be running**
+
+## How does it work
+
+DeviceManager stores and retrieves information models for devices and templates and a few static information about them as well. Whenever a device is created, removed or just edited, it will publish a message through Kafka. It depends only on DataBroker and Kafka.
+All messages published by Device Manager to Kafka can be seen in [Device Manager Messages](https://dojotdocs.readthedocs.io/projects/DeviceManager/en/latest/kafka-messages.html).
 
 ## Dependencies
 
 DeviceManager has the following dependencies:
 
-- flask (including flask_sqlalchemy)
+- flask (including flask_sqlalchemy, flask_migrate and flask_alembic)
 - psycopg2
 - marshmallow
 - requests
 - gunicorn
 - gevent
 - json-logging-py
 - kakfa-python
+- redis
 
 But you won't need to worry about installing any of these - they are automatically installed when starting DeviceManager.
-There must be, though, a postgres instance accessible by DeviceManager.
+
+## Configuration
+
+These are the environment variables used by device-manager
+
+Key                     | Purpose                                             | Default Value
+----------------------- | --------------------------------------------------- | --------------
+DBNAME                  | postgres database name                              | dojot_devm
+DBHOST                  | postgres database host                              | postgres
+DBUSER                  | postgres database user                              | postgres
+DBPASS                  | postgres database password                          | none
+DBDRIVER                | postgres database driver                            | postgresql+psycopg2
+CREATE_DB               | option to create the database                       | True
+KAFKA_HOST              | kafka host                                          | kafka
+KAFKA_PORT              | kafka port                                          | 9092
+STATUS_TIMEOUT          | kafka timeout                                       | 5
+BROKER                  | kafka topic subject manager                         | http://data-broker
+REDIS_HOST              | redis host                                          | device-manager-redis
+REDIS_PORT              | redis port                                          | 6379
+SUBJECT                 | subject of device-manager                           | dojot.device-manager.device
+DEV_MNGR_CRYPTO_PASS    | password of crypto                                  | none
+DEV_MNGR_CRYPTO_IV      | inicialization vector of crypto                     | none
+DEV_MNGR_CRYPTO_SALT    | salt of crypto                                      | none
```

Comment:
There is also the DEVICE_SUBJECT variable, but it doesn't seem to be used anywhere. 
I believe it is ideal to remove it

Reply:
Ok. I'll remove it.